README.md

```diff
@@ -1,16 +1,47 @@
 # Prolific Swift Style Guide
 
-This is Prolific's style guide for writing code using Swift.
+## Table Of Contents ##
 
-The purpose of this guide is to develop a universal standard for Swift code that makes our codebases
-consistent and easy to read.
+* [Overview](#overview)
+* **Standards**
+	* [File Structure](#file-structure)
+	* [Statement Termintaion](#statement-termination)
+	* [Variable Declaration](#variable-declaration)
+	* [Self](#self)
+	* [Bracket Syntax](#bracket-syntax)
+	* [Force Unwrap](#force-unwrap)
+	* [Access Modifiers](#access-modifiers)
+	* [Type Declarations](#type-declarations)
+	* [Nil Checking](#nil-checking)
+	* [Enums](#enums)
+* [Best Practices](BestPractices.md)
 
-This is a work in progress.
-If there is anything you are looking for that is not covered here you should refer to [Github's style guide](https://github.com/github/swift-style-guide).
 
-[Link](BestPractices.md) for Prolific's Swift Best Practices document.
+## Overview ##
```

Comment:
When you look at the README it feels that everything falls under Overview. Is that intended? Shouldn't the rules be under a `## Style Guide##` section?